OxfordBodleian/Juel-Jensen/BDLaethe33.xml

```diff
@@ -10,15 +10,15 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
     <teiHeader xml:base="https://betamasaheft.eu/">
         <fileDesc>
             <titleStmt>
-                <title xml:lang="gez" xml:id="title1">መዝሙረ፡ ዳዊት፡</title>
-                <title xml:lang="gez" corresp="#title1" type="normalized">Mazmura Dāwit</title>
-                <title xml:lang="en" corresp="#title1">Psalms of David</title>
+                <title xml:lang="gez" xml:id="title1">ስብሐተ፡ ፍቁር፡, መዝሙረ፡ ዳዊት፡</title>
+                <title xml:lang="gez" corresp="#title1" type="normalized">Sǝbḥāta fǝqur, Mazmura Dāwit</title>
+                <title xml:lang="en" corresp="#title1">Praise of the Beloved, Psalms of David</title>
```

Comment:
For me, this would be a case where we need to include the Geez in the English, because no one will understand what "Praise of the Beloved" is, except those who can translate it backwards into Geez :) Maybe "The prayer Sǝbḥata fǝqur <!-- Sǝbḥata fǝqur in italics -->"?

Reply:
What about if we give - for consistency - all English titles and add the transliterated titles in brackets for those cases that we feel are difficult for the reader to understand?
So, for example: "Praise of the Beloved (_Sǝbḥāta fǝqur_), Psalms of David". I just wonder how to do this without adding parts in Ge'ez in the English title

Reply:
That sounds perfect to me, but I don't know how to encode it in a sensible way. Could it be a comment in the xml and we add it manually at the same time as we add the italics?

Reply:
What do you think @thea-m ?
We might keep discussing this in the next meeting, but at the same time I would like not to take too much time away from the other issues

Reply:
I have now found a possibility to add mark up for italics already at this stage, which should really make things easier for us! We can use `<hi rendition="simple:italic">` such cases. Therefore, I would suggest that we focus on what will make the PDF production easy for us:
This would be one single title element in all records in which the form of the title is reproduced exactly as we want it to be printed, only using Latin characters and avoiding all markup except `<hi rendition="simple:italic">` for words we would like to appear in italics. 
For this example, that would look like:
`<title xml:lang="en">Praise of the Beloved (<hi rendition="simple:italic">Sǝbḥāta fǝqur</hi>), Psalms of David</title>`
What do you think?

Reply:
I agree with this solution. I am going to mark up all portions of text we want to be printed in italics and remove all comments

Reply:
You already did it Thea, I hadn't see. Thank you!